apa.csl

```diff
@@ -129,7 +129,7 @@
       <else>
         <choose>
           <if variable="DOI">
-            <text variable="DOI" prefix="doi:"/>
+            <text variable="DOI" prefix="http://dx.doi.org/"/>
```

Comment:
I asked the APA Style Experts. They basically said it didn't matter but that the technically correct version would be "dx." I think we can safely just do "doi.org".